hack/e2e.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash -e
+#
+# This script is intended to be run from the CI pipeline as follows:
+#
+#   # Define secrets in the envionment
+#   source hack/e2e.sh
+#   aro_ci_setup
+#   make e2e
+
+get_random_location() {
+    array[0]="australiasoutheast"
+    array[1]="eastus"
+    array[2]="westeurope"
+
+    size=${#array[@]}
+    index=$(($RANDOM % $size))
+    echo ${array[$index]}
+}
```

Comment:
can leave this for now but I'm not convinced - a) hard codes regions in the script, b) this kind of thing has caused us problems in v3